src/lib/prof-lean/hpcrun-fmt.h

```diff
@@ -171,7 +171,11 @@ hpcrun_fmt_hdr_free(hpcrun_fmt_hdr_t* hdr, hpcfmt_free_fn dealloc);
 #define HPCRUN_FMT_METRIC_SHOW_EXCLUSIVE  3
 #define HPCRUN_FMT_METRIC_INVISIBLE       4
 
-
+#define HPCRUN_FMT_DUMMY_NODE 65534
+#define HPCRUN_FMT_GPU_RANGE_ROOT_NODE 65533
+#define HPCRUN_FMT_GPU_RANGE_NODE 65532
+#define HPCRUN_FMT_GPU_CONTEXT_NODE 65531
+#define HPCRUN_FMT_GPU_STREAM_NODE 65530
```

Comment:
With these additions, please remove the similar `#defines` from `lib/profile/hpcrun4.cpp`.

Reply:
This most definitely hasn't been resolved yet.

Reply:
@blue42u You said **remove** similar defines from `lib/profile/hpcrun4.cpp`. I don't think hpcrun4.cpp still has the same definitions now.

Reply:
You are right, I was looking at the wrong branch when checking this. Sorry for the noise.